### ISSUE TITLE
Incompatibility with Lithium (without specific config edits)

### ISSUE THREAD

**Issue by Tkain** [user11681/paradox-slabs#1]
When Lithium is installed as normal, the slab-breaking feature of Paradox Slabs doesn't. The bounding box of the slabs in a double slab block aren't separate, and clicking on one of the slabs only breaks the double slab as a whole.

(To reproduce: Install Lithium and Paradox Slabs in a Minecraft environment. Load a new world, place two slabs down on top of each other, and observe.)

From my troubleshooting, this is fixed by adding `mixin.shapes.lazy_shape_context=false` to Lithium's configuration file, so it must have to do with whatever mixins that setting affects. 

For now, I'd recommend disabling the setting automatically within Paradox Slabs's metadata with `"custom": {"lithium:options": {"mixin.shapes.lazy_shape_context": false}}`, but if you have the time to do so, the most ideal solution would be to work with Lithium's changes (presumably using the Lithium API).

**Comment by user11681** [comment]
For now I have used a mixin, though ideally I would like to at least have `LithiumEntityShapeContext` extend `EntityShapeContext` (if not modified directly) so that my duck interface is applied to it automatically.

**Comment by user11681** [comment]
I have submitted an issue (jellysquid3/lithium-fabric#208) regarding this.

**Comment by Abbanon** [comment]
Just wanted to let you know, that a log error still persists in regards to this issue.
I'm running 1.16.5, using latest ParadoxSlabs (0.1.0) and latest Lithium (0.6.4).
An error appears in my log with both mods enabled. The error is such:
`[main/WARN]: Error loading class: me/jellysquid/mods/lithium/common/block/LithiumEntityShapeContext (java.lang.ClassNotFoundException: me/jellysquid/mods/lithium/common/block/LithiumEntityShapeContext)`

Is this error bad? Or is it just a harmless fallback message when trying to find/read older Lithium versions?

I've consulted with the CaffeineMC team, and it looks like this entirely on ParadoxSlabs' end, because the issue on Lithium's end has already been resolved.
